test/testdata/infer/fast_big_lubs.rb

```diff
@@ -0,0 +1,533 @@
+# typed: true
```

Comment:
Could you move this file into `test/testdata/perf`? We have a few other performance regression tests there already.

Reply:
Good idea; can do